Makefile

```diff
@@ -128,7 +128,8 @@ ifndef PULL_SECRET_FILE
 	PULL_SECRET_FILE = --pull-secret-file=$(HOME)/Downloads/crc-pull-secret
 endif
 ifndef BUNDLE_LOCATION
-	BUNDLE_LOCATION = --bundle-location=$(HOME)/Downloads/crc_libvirt_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
+	BUNDLE_LOCATION = --bundle-location=embedded
+##$(HOME)/Downloads/crc_libvirt_$(BUNDLE_VERSION).$(BUNDLE_EXTENSION)
```

Comment:
This comment can be removed I think? I see this is reverted in the next commit, so not sure what is intended here?

Reply:
@cfergeau that's my mistake. I had to change that for testing (changes only apply to embedded bundles). But I only checked if everything was back to non-embedded after the last commit. Hence the back and forth. Sorry, I'll address this.